cla_backend/apps/cla_eventlog/tests/test_find_and_delete_old_cases.py

```diff
@@ -0,0 +1,196 @@
+from django.test import TestCase
+from django.utils import timezone
+from freezegun import freeze_time
+import datetime
+
+from core.tests.mommy_utils import make_recipe
+from cla_eventlog.models import Log
+from complaints.models import Complaint
+from legalaid.models import Case, EODDetails, EligibilityCheck, PersonalDetails
+from diagnosis.models import DiagnosisTraversal
+from cla_auditlog.models import AuditLog
+from cla_eventlog.management.commands.find_and_delete_old_cases import Command
+
+
+def _make_datetime(year=None, month=None, day=None, hour=0, minute=0, second=0):
+    today = datetime.date.today()
+    year = year if year else today.year
+    month = month if month else today.month
+    day = day if day else today.day
+    dt = datetime.datetime(year, month, day, hour, minute, second)
+    return timezone.make_aware(dt, timezone.get_current_timezone())
+
+
+class FindAndDeleteOldCases(TestCase):
+    def setUp(self):
+        super(FindAndDeleteOldCases, self).setUp()
+        self.instance = Command()
+
+    def create_case(self, current_time, case_type="legalaid.case"):
+        case = None
+        freezer = freeze_time(current_time)
+        freezer.start()
+        case = make_recipe(case_type)
+        freezer.stop()
+        return case
+
+    def create_event_log_for_case(self, case, code, created):
+        make_recipe("cla_eventlog.log", case=case, code=code, created=created)
+
+    def delete_old_cases(self, current_time):
+        self.instance.execute("delete")
+
+    def find_old_cases(self, current_time):
+        cases = None
+        freezer = freeze_time(current_time)
+        freezer.start()
+        # Using handle method because command execute method can only return a string i.e it can't return a queryset
```

Comment:
It won't work, but I thought that's fine since this is just for having it run in the tests

Reply:
Will it break in a confusing way?

Reply:
We are using `handle` here, so it's just calling the function and not executing it like a command. I don't think that should break for any reason?

Reply:
I mean if it were to be run on the command line

Reply:
This would break in command line. Is that a problem? Is the issue if we were to run this by accident otherwise I am not sure what's wrong

Reply:
Just generally good practice not to build things that would break if used in a particular way. It's not a 100% deal-breaker if it turns out to be unreasonably difficult, but I think you should be able to identify whether it's running from the command line or not.

Reply:
Done